assets/src/edit-story/components/mediaPicker/useMediaPicker.js

```diff
@@ -38,6 +37,7 @@ export default function useMediaPicker({
   buttonInsertText = __('Insert into page', 'web-stories'),
   onSelect = () => {},
   onClose = () => {},
+  onRestrictAccess = () => {},
```

Comment:
Maybe rename this prop to `onPerrmissionError` for clarity?

Reply:
```suggestion
  onPermissionError = () => {},
```

Reply:
Renamed.